### ISSUE TITLE
SC Content on Hover or Focus: Accessibility of Re-positioning

### ISSUE THREAD

**Issue by steverep** [w3c/wcag21#499]
This issue calls the accessibility of re-positioning into question.  The condition to make the trigger visible reads:

> **Visible Trigger:** Either the additional content does not obscure any essential content within the triggering user interface component, or the additional content can be closed or repositioned by the user;

First, there is a problem here because the screen extents of the re-positioning are not specified.  As an extreme, the ability to move the additional content just 1 pixel would pass.  The objective of this bullet in the criterion is to make sure the trigger can be viewed easily and entirely, and there is far from a guarantee that simply saying "re-positioning" as an option will accomplish this goal.

More importantly, the targeted beneficiaries here are people with low vision or low pointer accuracy, and re-positioning content is not generally an easily accessible feat for this group and simply cannot be done with precision.  This is especially true again when the objective is to view the trigger, which may not even be partially in view while trying to move content, making the task a trial and error loop.

It seems extremely odd to be allowing (and in a way advocating for) such a questionably accessible method of viewing the trigger.  It is recommended this option be removed.  Both not obscuring and/or the option to close are much easier to implement and much more accessible to the target beneficiaries.

**Comment by DavidMacDonald** [comment]
+1

**Comment by steverep** [comment]
From what I gathered from the minutes and remotely participating when this issue was [discussed at TPAC](https://www.w3.org/2017/11/06-ag-minutes.html#item05), some folks seemed to infer that I was arguing that re-positioning should not be allowed.  I hope I was able to clear up this misunderstanding.  In short, re-positioning is free for the author to go to the trouble to provide, but it should never be considered an accessible path to viewing the trigger by itself, i.e. a situation where anytime I hover or focus an element I need to move something out of the way to see it with no ability to close it.

The other big thing I took away from the discussion was that many folks thought the ability to dismiss was a critical feature, more so than not  obscuring the trigger.  From a low vision standpoint, they are about equal with dismissing having the extra advantage of helping to see the content surrounding the trigger.  However, from a keyboard user standpoint in the case of components like mega menus, dismissal has the only advantage and could save quite a bit of completely unnecessary tabbing.  This is what led to the current proposal in pull request #569 by the LVTF, and would obviously clear this issue as well.
